dencam.py

```diff
@@ -39,27 +39,28 @@
     configs = yaml.load(f, Loader=yaml.SafeLoader)
 log.info('Read in configuration settings')
 
-NUM_STATES = 5
-
 
 def main():
 
     flags = {'stop_buttons_flag': False}
+    STATE_LIST = ['OffPage', 'NetworkPage', 'RecordingPage', "SolarPage", "BlankPage"]
 
     def cleanup(flags):
         flags['stop_buttons_flag'] = True
         time.sleep(.1)
 
     try:
         recorder = Recorder(configs)
-        state = State(NUM_STATES)
```

Comment:
I am confused by the use of the State constructor. Is this a class we are using? Is there some benefit to this over state being simply an integer since you only seem to access state.value?

Reply:
This line is gone but the use of the State class remains so I'll chime in.  Firstly, the State class wasn't introduced by this PR (it was introduced by me :-) ) so its use/removal is a broader discussion. But I do think Kyra makes a good point that maybe it is overkill. 

It was partially intended as placeholder for a Finite State Machine that is more complicated when/if the UI doesn't just simply cycle through a fixed state order. Not sure we'll ever need that at this point, though.  It does encapsulate things a little bit still, however, and does automatically loop around so it also encapsulates that mechanic, although potentially that could be accomplished just as well with an itertools.cycle.  What I suggest is that if we think we should pursue this, that it get written up as a separate issue where we can follow through on a discussion about it but that for the purposed of this PR we let it remain.